configs.go

```diff
@@ -1164,6 +1164,7 @@ type WebhookConfig struct {
 	MaxConnections     int
 	AllowedUpdates     []string
 	DropPendingUpdates bool
+	SecretToken        string
```

Comment:
I think, adding the parameter just to the webhook config isn't enough. Verification of the token on getting a webhook is also needed. Otherwise, there is no sense in setting the token if it isn't checked afterwards.

Reply:
This is an API call to `setWebhook`. This is enough to add that parameter. Of course, you also have to check for it in `X-Telegram-Bot-Api-Secret-Token`, but that's in a different part of the application, and can be done on the application layer, not inside this library.

Reply:
Actually, looks like @efulmo was right. In `bot.go` there is a method `ListenForWebhook(string)`, which will need to verify the token.